new-releases-systembolaget/systemet.py

```diff
@@ -0,0 +1,32 @@
+# post to slack everytime a new excel finle with beer releases on systembolaget is avaliable during 2018
+# -*- coding: utf-8 -*-
+import csv
+import json
+import requests
+systembolaget = 'https://www.systembolaget.se/fakta-och-nyheter/nyheter-i-sortimentet/lanseringar/'
+url = ''  # input slack webhook
+release_dates = ['19 jan', '2 feb', '16 feb', '2 mars', '16 mars', '6 april', '20 april', '4 maj', '18 maj', '8 juni', '6 juli', '3 aug', '17 aug', '7 sep', '21 sep', '5 okt', '19 okt', '2 nov', '16 nov', '7 dec']
+
+with open('sma_partier.csv', 'r', encoding='utf-8') as csv_file:
```

Comment:
Maybe should be named "2018_sma_partier.csv" and check for current year. That way it just keeps going the years to come.